### PR TITLE
Add boring-cyborg

### PR DESCRIPTION
**Pull request by pankajastro**
Just in case we want to add encouraging comments for 1st contributions

### CODE REVIEW

**Review by Lee-W — APPROVED** [review]
Do we need to install this APP to the repo before we merge it?

**Comment by pankajastro** [comment]
> Do we need to install this APP to the repo before we merge it?

I think, yes, installations require. I wait for @phanikumv to take a call on this.

**Comment by Lee-W** [comment]
Got it. Sounds good 👍

File: .github/boring-cyborg.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+firstPRWelcomeComment: >
+  Thanks for opening this pull request! Please check out our contributing guidelines.
+
+# Comment to be posted to congratulate user on their first merged PR
+firstPRMergeComment: >
+  Awesome work, congrats on your first merged pull request!
+
+# Comment to be posted to on first time issues
+firstIssueWelcomeComment: >
+  Thanks for opening your first issue here! Be sure to follow the issue template!
```
Comment (pankajkoti) [inline .github/boring-cyborg.yml:13]:
```suggestion
  Thanks for opening your first issue here! Make sure to follow the issue template!
```

File: .github/boring-cyborg.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+##### Greetings ###########################################################################
+# Comment to be posted to welcome users when they open their first PR
+firstPRWelcomeComment: >
+  Thanks for opening this pull request! Please check out our contributing guidelines.
```
Comment (pankajkoti) [inline .github/boring-cyborg.yml:5]: Can we add a link to the contributing guidelines?

**Pull request closed without merging**